Documentation/api.md

```diff
@@ -238,3 +238,16 @@ matchbox.foo/assets/
         ├── coreos_production_pxe.vmlinuz
         └── coreos_production_pxe_image.cpio.gz
 ```
+
+
+# HTTPS - The matchbox API is also available with HTTPS
+
+To start matchbox in this mode you will need the following flags set:
+
+| Name        | Type   | Description                                                   |
+|-------------|--------|---------------------------------------------------------------|
+| sslEnabled  | bool   | true/false                                                    |
```

Comment:
all these should use the name and style that is available on the command line - `-web-ssl`, `-web-cert-file`, and `-web-key-file`, and it doesn't look like the `caFile` is actually relevant for enabling https, only for setting up the grpc tls.